src/structures/GuildMember.js

```diff
@@ -159,6 +171,15 @@ class GuildMember extends Base {
     return this.joinedTimestamp ? new Date(this.joinedTimestamp) : null;
   }
 
+  /**
+   * The time this member's timeout will be removed
+   * @type {?Date}
+   * @readonly
+   */
+  get communicationDisabledUntil() {
+    return this.communicationDisabledUntilTimestamp && new Date(this.communicationDisabledUntilTimestamp);
```

Comment:
Do we usually do it like this?

Reply:
Currently, we do a ternary: `property ? new Date(property) : null`. However, as of #7108, it will change to what is here.